.travis.yml

```diff
@@ -24,6 +24,7 @@ branches:
   only:
     - master
     - develop
+    - feature/ats-707_core_aio
```

Comment:
Note: this should not be merged as-is (hard-coded branch name).

Reply:
Yup - standard procedure